app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -341,6 +338,20 @@ public Survey getSurveyMostRecentlyPublishedVersion(StudyIdentifier studyIdentif
         return new QueryBuilder().setStudy(studyIdentifier).isPublished().setSurvey(guid).isNotDeleted().getOne(true);
     }
     
+    // This is a slower call. There's no getting around it. But not as slow as calling all of these separately
+    // from the Researcher UI.
+    // secondary index query (via getAllSurveysMostRecentlyPublishedVersion)
+    @Override
+    public List<Survey> getSurveysSummary(StudyIdentifier studyIdentifier) {
+        List<Survey> surveys = getAllSurveysMostRecentlyPublishedVersion(studyIdentifier);
+        // get the questions for each survey from the whole survey, then copy over only the questions
+        for (int i=0; i < surveys.size(); i++) {
+            Survey updated = getSurvey(surveys.get(i));
+            surveys.get(i).setElements(ImmutableList.copyOf(updated.getUnmodifiableQuestionList()));
```

Comment:
getUnmodifiableQuestionList already returns an unmodifiable list. Wrapping it in another ImmutableList.copyOf() is redundant (even if it's optimized by Guava).